mappings/net/minecraft/client/util/GLFWUtil.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS cnj net/minecraft/client/util/GLFWUtil
+	METHOD b getGLFWTime ()D
```

Comment:
Why not just getTime? It's in GLFWUtil.

Reply:
Should this also specify the unit of time?

Reply:
I've renamed it to getTime but specifying the unit of time should be in a javadoc since it is platform dependent and specific to the GLFW timer.